### PR TITLE
[Calling][Bugfix] String Difference in SnackBar

### PR DESCRIPTION
**Pull request by inancyGit**
=> Different messages depending on platform when Client completely lost connection and didn't reconnect Changed string in ios.

## Purpose

* ...

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [x] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull request checklist

This PR has considered:
- [ ] Color Theming
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] View Data Injection
- [ ] Demo App UI/UX update

## How to Test

* Open...
* Click on...

## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### CODE REVIEW

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -128,7 +128,7 @@
 "AzureCommunicationUICalling.SnackBar.Button.Dismiss.AccessibilityLabel"="Dismiss Banner";
 "AzureCommunicationUICalling.SnackBar.Button.Dismiss.AccessibilityHint"="Dismisses this notification";
 "AzureCommunicationUICalling.SnackBar.Text.ErrorCallJoin"="Unable to join the call due to an error.";
-"AzureCommunicationUICalling.SnackBar.Text.ErrorCallEnd"="You were removed from the call due to an error.";
+"AzureCommunicationUICalling.SnackBar.Text.ErrorCallEnd"="You may have been removed from the meeting, or lost your connection. Try to join again.";
```
Comment (pavelprystinka) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en.lproj/Localizable.strings:131]: We have `ErrorCallEvicted` for the `may have been removed` - do you get ErrorCallEnd message when user is removed from the call? If so, it is not correct.
Can we distinguish when call is disconnected due to network issues? Could you please check what we receive in the call end reason from the SDK (there is code and subcode).
`ErrorCallEnd` should be reserved to the unknown call end reason only.

**Review by pavelprystinka — CHANGES_REQUESTED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (inancyGit) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en.lproj/Localizable.strings:131]: The code which I get in log is : **_Subscription cancelled with Error Code: callEndFailed_**  and for CallEvicted: This is the message 
"AzureCommunicationUICalling.SnackBar.Text.ErrorCallEvicted"="You were removed from the call.";

Comment (pavelprystinka) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en.lproj/Localizable.strings:131]: Discussed:
"You were removed from the call due to an error or your lost connection. Try to join again."

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en-GB.lproj/Localizable.strings [hunk 2]
```diff
@@ -128,7 +128,7 @@
 "AzureCommunicationUICalling.SnackBar.Button.Dismiss.AccessibilityLabel"="Dismiss Banner";
 "AzureCommunicationUICalling.SnackBar.Button.Dismiss.AccessibilityHint"="Dismisses this notification";
 "AzureCommunicationUICalling.SnackBar.Text.ErrorCallJoin"="Unable to join the call due to an error.";
-"AzureCommunicationUICalling.SnackBar.Text.ErrorCallEnd"="You were removed from the call due to an error.";
+"AzureCommunicationUICalling.SnackBar.Text.ErrorCallEnd"="You may have been removed from the meeting, or lost your connection. Try to join again.";
```
Comment (pavelprystinka) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en-GB.lproj/Localizable.strings:131]: Please align it with Android message.

**Review by pavelprystinka — CHANGES_REQUESTED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en-GB.lproj/Localizable.strings (hunk 2 above)
Comment (inancyGit) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Localization/en-GB.lproj/Localizable.strings:131]: @pavelprystinka  I aligned it in the file Localizable.strings(English) It is generic one. Hence, it reflects there.

**Review by iaulakh — DISMISSED** [review]

**Review by Mohtasim — DISMISSED** [review]

**Review by ShaunaSong — APPROVED** [review]

**Review by Mohtasim — APPROVED** [review]

**Review by pavelprystinka — APPROVED** [review]

**Pull request merged**